.github/workflows/ci.yml

```
@@ -137,6 +138,16 @@ jobs:
             fi
           fi
 
+      - name: Restore cache
+        if: github.event_name == 'pull_request' && github.event.pull_request.draft == false
+        id: cache-restore
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)